### PR TITLE
add getWinner script and command. finish week2 report

### PR DESCRIPTION
**Pull request by AndyOooh**
## Description
Added script and command  for `getWinner`. It covers two Ballot.sol functions: `winningProposal` and `winnerName`
Also turned off `no-console` for eslint.

Edit:
Added a commit with updates to readme. and deploy command. More details in commit message.

## Motivation and Context
Finish week2 project.

## How Has This Been Tested?
Ran the command and logged results to console.
Results were surprising as the winner was proposal _milk_ with only 1 vote, whereas 5 votes were cast for two proposals in total.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Style (style only changes)
- [x] Refactor (code that does not add new functionality nor fixes a bug)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
         "import/no-named-as-default": "off",
         "import/no-unresolved": "warn",
         "no-mixed-spaces-and-tabs": 0,
-        "no-console": 2,
+        "no-console": 0,
```
Comment (evchip) [inline .eslintrc.json]: The number is for how many the linter will accept. If you want to turn it off, change `0` to `"off"`

File: scripts/transactions/getWinner.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import {ethers} from 'hardhat';
+import getBallotContract from '../utils/getBallotContract';
+
+dotenv.config();
+
+async function getwinningProposal(contractAddress: string): Promise<void> {
+  const ballot = getBallotContract(contractAddress);
+  try {
+    const winningProposalVoteCount = await ballot.winningProposal();
+    const winningProposalName = await ballot.winnerName();
```
Comment (evchip) [inline scripts/transactions/getWinner.ts]: Best to wrap each promise in its own `try/catch`. Helps to trace errors more easily

**Review by evchip — APPROVED** [review]
A few minor comments. Overall looks good.

File: .eslintrc.json (hunk 1 above)
Comment (AndyOooh) [reply .eslintrc.json]: oh, ok

File: scripts/transactions/getWinner.ts (hunk 2 above)
Comment (AndyOooh) [reply scripts/transactions/getWinner.ts]: alright

**Comment by AndyOooh** [comment]
Implemented requested changes. Gonna merge now to avoid someone seeing an unfinished report.

**Pull request merged**